oeps/processes/oep-0021-proc-deprecation.rst

```diff
@@ -286,25 +331,28 @@ Here are some common ways to mark a technology as deprecated:
 
 * REST API - Specify in the 1st line of the API's docstring::
 
-    "Deprecated <DEPR-Number>"
+    "Deprecated <link-to-gh-issue>"
 
 * Javascript code - Add a log statement that executes once without being noisy::
 
-    console.log("<Technology name> is deprecated. See <DEPR-Number>.")
+    console.log("<Technology name> is deprecated. See <link-to-gh-issue>.")
 
 * Python code - Add a warnings.warn_ statement so it executes once without being noisy::
 
-    warnings.warn("<Technology name> is deprecated. See <DEPR-Number>.", DeprecationWarning)
+    warnings.warn("<Technology name> is deprecated. See <link-to-gh-issue>.", DeprecationWarning)
 
 * Feature toggles - Set the “Expiration Date” as described in OEP-17_.
 
-* Github repo - See `OEP-14 <https://open-edx-proposals.readthedocs.io/en/latest/oep-0014-proc-archive-repos.html>`_
+* GitHub repo - See `OEP-14 Archiving Open edX GitHub Repositories`_.
+
+* XBlock - For edx.org specifically, see `Deprecating xBlock for the edX website`_ (internal edX use only).
 
-* XBlock - For edx.org specifically, see `Deprecating xBlock for the edX website`_.
+**For Code Review: The above link is specific to edX and not externally visible. Should it
+really be part of an OEP? If so, should other community members get to insert their own process into OEPs as well?**
```

Comment:
Code reviewers: Please advise

Reply:
I'm okay removing this link and moving it to just internal 2U/edX documentation.

Reply:
Done. Will you take care of finding the proper place for it?

Reply:
The page is now here: https://openedx.atlassian.net/wiki/spaces/COMM/pages/151912534/Proposed+Github+Deprecation+Process

Reply:
I don't see that referenced anywhere in this doc (https://open-edx-proposals.readthedocs.io/en/latest/processes/oep-0021-proc-deprecation.html)

The comment I wrote referred to this doc: https://openedx.atlassian.net/wiki/spaces/ENG/pages/723550424/Deprecating+and+Disabling+an+XBlock+for+the+edX+website

I could see keepi ng that in as a reference to other site operators, but the page would need to be public.